book/030-tutorial-downstream/060-alpha-diversity.md

```diff
@@ -11,3 +11,165 @@ name: tutorial
+effects are some (ostensibly random) per-group coefficient which minimizes the
+error within that group. In our situation, we would want our random effect to
+be the `PatientID` as we can see each subject has a different baseline for
+diversity (and we have multiple measures for each patient).
+By making that a random effect, we can more accurately ascribe associations to
+the fixed effects as we treat each sample as a "draw" from a per-group
+distribution.
+
+There are several ways to create a linear model with random effects, but we
+will be using a *random-intercept*, which allows for the per-subject intercept
+to take on a different average from the population intercept (modeling what we
+saw in the group-significance plot above).
+
+````{margin}
+```{note}
+It is called a random effect because the cause of the per-subject deviation
+from the population intercept is not known. It's source is "random" in the
+statistical sense. That is randomness is not introduced to the model, but is
+instead tolerated by it. Additionally, unlike fixed-effects, there is no
+particular meaning in the random-effect's estimate (and are therefore not
+reported). After all, how would we ascribe any particular association to the
+unknown?
+```
+````
+
+First let's evaluate the general trend of the transplant.
+
+```{usage}
+
+md_observed_features = use.view_as_metadata(
+    'md_observed_features',
+     core_metrics_results.observed_features_vector)
+
+merged_features = use.merge_metadata(
+    'merged_features',
+    sample_metadata,
+    md_observed_features)
+
+use.action(
+    use.UsageAction('longitudinal', 'linear_mixed_effects'),
+    use.UsageInputs(metadata=merged_features,
+                    state_column='DayRelativeToNearestHCT',
+                    individual_id_column='PatientID',
+                    metric='observed_features'),
+    use.UsageOutputNames(visualization='lme-obs-features-HCT')
+)
+
+```
+
+Here we see a significant association between richness and to the bone marrow
+transplant.
+
+We may also be interested in the effect of the auto fecal microbiota
+transplant. It should be known that these are generally correlated, so choosing
+one model over the other will require external knowledge.
+
+
+```{usage}
+use.action(
+    use.UsageAction('longitudinal', 'linear_mixed_effects'),
+    use.UsageInputs(metadata=merged_features,
+                    state_column='day-relative-to-fmt',
+                    individual_id_column='PatientID',
+                    metric='observed_features'),
+    use.UsageOutputNames(visualization='lme-obs-features-FMT')
+)
+```
+
+We also see a downward trend from the FMT. Since the goal of the FMT was to
+ameliorate the impact of the bone marrow transplant, and the timing of the FMT
+is related to the timing of the marrow transplant, we might deduce that the
+negative coefficient is primarily related to the bone marrow transplant and is
```

Comment:
It's actually most likely the extreme antibiotic treatment they receive pre- and peri-HCT. So you could say "primarily related to the bone marrow transplant procedure" (so it's referring to the whole treatment, rather than just the transplant itself), or get more specific if you want to.